plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -1862,5 +1862,55 @@ function ( $collection_args, $query, $request ) {
 				return $collection_args;
 			}
 		);
+
+		$this->register_collection_handlers(
+			'woocommerce/product-collection/upsells',
+			function ( $collection_args ) {
+				// No products should be shown if no related product reference is set.
+				if ( empty( $collection_args['upsellsProductReference'] ) ) {
+					return array(
+						'post__in' => array( -1 ),
+					);
+				}
+
+				$product = wc_get_product( $collection_args['upsellsProductReference'] );
+				$upsells = array_map(
+					function ( $id ) {
+						return strval( $id );
+					},
+					$product->get_upsell_ids()
+				);
```

Comment:
```suggestion
				$upsells = array_map( 'strval', $product->get_upsell_ids() );
```

Interestingly, PHP actually lets you just use the string name of the function as the argument!